examples/diagnostics_and_criticism/sampler-stats.ipynb

```diff
@@ -14,58 +14,56 @@
   },
```

Comment:
<p>I added an issue about this, but I think we can do that manually in the meantime, also to show one can customize the InferenceData with <code>idata_kwargs</code>.  In this particular case, I'd use:</p><p> </p><pre class="ql-syntax" spellcheck="false">coords = {"step": ["BinaryMetropolis", "Metropolis"]}
dims = {"accept": ["step"]}
...
   trace = pm.sample(..., idata_kwargs={"dims": dims, "coords": coords})
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/94/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>